src/cadenza.js

```diff
@@ -587,7 +589,13 @@ export class CadenzaClient {
       useMapSrs,
       operationMode,
     });
-    return this.#show(resolvePath(backgroundMapView), params, signal);
+    console.log('createGeometry');
+    await this.#show(resolvePath(backgroundMapView), params, signal);
+    if (additionalLayers) {
+      additionalLayers.forEach((layer) =>
+        this.#postEvent('importLayers', layer),
```

Comment:
(Needs resolution)
Since the event is called `importLayers` (plural), I'd expect it so be able to import multiple layers at the same time, no?

Reply:
I'll change it to importLayer